applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -7,8 +7,9 @@ Power manager module
    :local:
    :depth: 2
 
-Use the power manager module to reduce the energy consumption of battery-powered devices.
+Use the Power manager module to reduce the energy consumption of battery-powered devices.
```

Comment:
Is there a convention how to start module names - with capital or lower letter? 
As i can see, in this section we have it starting from capital letter but in `Configuration` section it is starting from lower letter. 
In `settings_loader.rst:10-11` there is similar case.

Reply:
Changed to lowercase